src/lib/services/index.ts

```diff
@@ -125,6 +126,13 @@ export const scheduleServices = async (
         ),
         secondsToMilliseconds(1),
     );
+
+    schedulerService.schedule(
+        featureToggleService.markPotentiallyStaleFeatures.bind(
+            featureToggleService,
+        ),
+        secondsToMilliseconds(10),
```

Comment:
This might be a little excessive? I understand we want our event to be as accurate as possible, but perhaps once a minute?

Reply:
Absolutely! It was just something I put in for now. Thanks for picking up on it! I agree that once a minute should be more than enough.